src/legacy/core_plugins/kibana/public/management/index.scss

```diff
@@ -7,9 +7,7 @@
 //   mgtChart__legend--small
 //   mgtChart__legend-isLoading
 
-@import 'hacks';
```

Comment:
Are you sure the whole `hacks` file import should be removed? The file still seems to exist and have styles in it.

Reply:
I checked and didn't find any places where these styles used. I deleted this file.